yt_dlp/extractor/xiaohongsu.py

```diff
@@ -0,0 +1,88 @@
+from .common import InfoExtractor
+from ..utils import (
+    float_or_none,
+    js_to_json,
+    merge_dicts,
+    url_or_none,
+)
+from ..utils.traversal import traverse_obj
+
+
+class XiaoHongSuIE(InfoExtractor):
+    _VALID_URL = r'https?://www\.xiaohongshu.com/explore/(?P<id>[a-f0-9]+)'
+    IE_DESC = '小红书'
+    _TESTS = [{
+        'url': 'https://www.xiaohongshu.com/explore/6411cf99000000001300b6d9',
+        'info_dict': {
```

Comment:
md5

Reply:
Resolved in c895dfea107b75db9d8873c7a90037a7b4c247ca